scripts/run-integration-tests.sh

```diff
@@ -211,6 +216,22 @@ if [[ $TEST_PASS -eq 0 && "$RUN_CONFORMANCE" == true ]]; then
   echo "TIMELINE: Conformance tests took $CONFORMANCE_DURATION seconds."
 fi
 
+if [[ "$RUN_PERFORMANCE_TESTS" == true ]]; then
+    START=$SECONDS
+    $KUBECTL_PATH apply -f deploy-130-pods.yaml
+    run_performance_test_130_pods
+    $KUBECTL_PATH delete -f deploy-130-pods.yaml
+
+    $KUBECTL_PATH apply -f deploy-730-pods.yaml
+    run_performance_test_730_pods
+    $KUBECTL_PATH delete -f deploy-730-pods.yaml
+
+    $KUBECTL_PATH apply -f deploy-5000-pods.yaml
+    run_performance_test_5000_pods
+    $KUBECTL_PATH delete -f deploy-5000-pods.yaml
+    PERFORMANCE_DURATION=$((SECONDS - START))
```

Comment:
I think we want to keep these timings separate, so 3 distinct values. One is pod density (730 pods on one node), and the 5000 on 100 nodes is more cluster size and EC2 calls.

Reply:
I did record per-test durations in the common.sh functions. Those numbers are printed when the test finishes and then uploaded to s3. But I could also print them at the end of the script if you think that'd be helpful!

Reply:
Hmm, maybe. This is just the cluster delete time anyway. 😄